src/integrationTest/java/uk/gov/hmcts/reform/userprofileapi/integration/UpdateUserProfileIntTest.java

```diff
@@ -208,6 +208,36 @@ public void should_return_400_when_any_mandatory_field_missing() throws Exceptio
 
     }
 
+    @Test
+    public void should_return_400_with_invalid_first_name() throws Exception {
+        UserProfile persistedUserProfile = userProfileMap.get("user");
+        String idamId = persistedUserProfile.getIdamId();
+
+        UpdateUserProfileData data = buildUpdateUserProfileData();
+        data.setFirstName("&this!is>invalid<");
+
+        userProfileRequestHandlerTest.sendPut(
```

Comment:
try adding in same test(functional and integration tics both) first name as per JIRA like 
"%3cscript%3ealert(%22WXSS%22)%3c%2fscript%3e". Don't create new test for this instead repeat code from line 219 to 223 with above first name in request

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-user-profile-api/pull/257/commits/c70410ae3d04305fb1b57a5ea3674124f69e8e25